docs/reference/aggregations/bucket/time-series-aggregation.asciidoc

```diff
@@ -9,6 +9,18 @@ preview::[]
 The time series aggregation queries data created using a time series index. This is typically data such as metrics
```

Comment:
consider adding the link in this paragraph.

if you're adding this info because the results aren't meaningful / it won't work unless the tsds structure is followed, you might want to reposition your note to say "if you don't use a TSDS, then xyz will happen"

```suggestion
The time series aggregation queries data created using a <<tsds,time series index>>. This is typically data such as metrics
```